dist/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To-Do-List</title>
+<script defer src="main.js"></script></head>
+
+<body>
+    <section class="main">
+        <header class="main-heading">
+            <h1>Faiz's To Do List</h1>
+        </header>
+        <div class="heading-list">
+
+        </div>
```

Comment:
- [ Optional ] I think it would be better if your `todo` list has an `input` field of type `text` where a user can enter `tasks` that they would like to add to the to-do list. I highly suggest that you do this because future milestones of this project will depend on that field.